exporter/honeycombexporter/honeycomb_test.go

```diff
@@ -188,80 +190,85 @@ func TestExporter(t *testing.T) {
 			},
 		},
 	}
-	got := testTraceExporter(td, t)
+	got := testTraceExporter(internaldata.OCToTraceData(td), t)
```

Comment:
I changed the order of events because the to/from OC changes the order of the Spans.